source/Halibut.Tests/Support/WebSocketSslCertificateHelper.cs

```diff
@@ -8,7 +8,7 @@ static class WebSocketSslCertificateHelper
     {
         internal static void AddSslCertToLocalStore()
         {
-            var store = new X509Store(StoreName.My, StoreLocation.LocalMachine);
+            var store = new X509Store(StoreName.My, StoreLocation.CurrentUser);
```

Comment:
Linux LocalMachine is read-only

Reply:
This change is adding support for websockets on linux?

Reply:
Ability to install the cert on Linux, we still have an issue where we can't run any tests due to pollingoverwebsockets Client not being able to bind to an SSL port on Linux